tools/tertiary-analysis/atlas-data-import/atlas_import_classifiers.xml

```diff
@@ -0,0 +1,29 @@
+<tool id="atlas_import_classifiers" name="Atlas import: get classifiers" version="@TOOL_VERSION@+galaxy0"  profile="@PROFILE@">
```

Comment:
I would suggest that we keep this in a separate branch. We cannot merge this and add it to the toolshed until classifier data is not a part of the Atlas SC release.